retry.go

```diff
@@ -166,3 +169,78 @@ func (b *Breaker) Failure(event Event, err error) bool {
 	b.last = time.Now().UTC()
 	return false // never drop events.
 }
+
+var (
+	// DefaultExponentialBackoffConfig provides a default configuration for
+	// exponential backoff.
+	DefaultExponentialBackoffConfig = ExponentialBackoffConfig{
+		Base:   time.Second,
+		Factor: time.Second,
+		Max:    20 * time.Second,
+	}
+)
+
+// ExponentialBackoffConfig configures backoff parameters.
+type ExponentialBackoffConfig struct {
+	// Base is the minimum bound for backing off after failure.
```

Comment:
Should probably make it clear that this is not actually a lower bound, but rather a lower bound on the size of the interval from which a delay is randomly chosen?